src/20250325_AaveV3EthereumLido_OrbitProgramRenewal/OrbitProgramRenewalData.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+library OrbitProgramRenewalData {
+  // stream information
+  uint256 public constant STREAM_DURATION = 90 days;
```

Comment:
In forum post and md, is specified as 80 days, but using 90 here. My guess is that 90 is the correct number as its a quarter?

Reply:
Yes 90 days is the right number (we put 80 because historicly we did a direct transfer worth of 10 days of stream worth + 80 days of stream, so we have "90 days")
But now that we remove the direct transfer (request from Marc), we should have put back 90.

But becase it has been voted like this, to avoid any conflicts, let's use 80 instead of 90 here.

Reply:
Finally we put back 90 days everywhere